src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,28 @@
+package duke.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
As per the Java coding standard, comments should be indented relative to their position in the code.
![image](https://user-images.githubusercontent.com/77711860/190870367-ae15d1a7-bff7-49a4-9a83-95ee4da4fa1e.png)